mtags/src/main/scala/scala/meta/internal/pc/Keyword.scala

```diff
@@ -20,7 +23,9 @@ case class Keyword(
     // Is this keyword only in Scala 3?
     isScala3: Boolean = false,
     // Optional character to select this completion item, for example "."
-    commitCharacter: Option[String] = None
+    commitCharacter: Option[String] = None,
+    // The (reverse) tokens that should appear before the given position (removing whitespace and EOF)
+    leadingReverseTokens: Option[Iterator[Token] => Boolean] = None
```

Comment:
1. I wanted to ask "won't it be better to move all of those comments to the proper scaladoc so they can be displayed in the hoover tooltip" but it turns out that this is not the case. I wonder if `apply` invocation can be treated similar to the direct constructor call.
Here is `apply`
<img width="860" alt="Screenshot 2022-09-21 at 19 45 26" src="https://user-images.githubusercontent.com/37124721/191574746-a3769f9f-078d-450d-a711-ef3765f43c3f.png">

and here is constructor call (after I moved one comment to the scaladoc)

<img width="898" alt="Screenshot 2022-09-21 at 19 46 31" src="https://user-images.githubusercontent.com/37124721/191574940-ef7dd7e5-db21-4cdc-ab5f-333c117743a3.png">

2. I find parameter name a bit misleading, it suggests that these are the reversed tokens, but in fact this is an optional predicate to test them. WDYT about `reversedTokensPredicate`?

Reply:
https://github.com/scalameta/metals/pull/4416/commits/bd431e031ef6dbe08abf58b5f61411226a72493b

Reply:
> Hmm, right, but it should show the scaladoc even for apply, I'll open an issue

I was counting on that ;)